conf/messages-ecc.en

```diff
@@ -24,7 +24,6 @@ eds.existing-eori.enrolment.confirmation.heading=Application complete
 eds.existing-eori.enrolment.confirmation.para1=You are now enrolled to the {0} service.
 
 cds.subscription.outcomes.feedback=What did you think of this service?
-cds.subscription.outcomes.feedback.hint=(takes 30 seconds)
```

Comment:
We have a ticket to add opening in the new tab/window for this link? Because instead of removing it it's probably worth to add opening in the new window.

In general we can still treat this as a separate stories.